pkg/oc/admin/diagnostics/config.go

```diff
@@ -32,7 +32,7 @@ func (o DiagnosticsOptions) detectClientConfig() (expected bool, detected bool)
 		return false, false
 	}
 	o.Logger().Notice("CED2011", "Determining if client configuration exists for client/cluster diagnostics")
-	confFlagName := config.OpenShiftConfigFlagName
```

Comment:
this looks breaking.  They're doing odd things and you have to tolerate both right?  Add a test that would have caught this.

Reply:
@sosiouxme I'll defer to you. We're trying to switch from `--config` to `--kubeconfig` and deprecated the former, does this fix look right?

Reply:
> @sosiouxme I'll defer to you. We're trying to switch from --config to --kubeconfig and deprecated the former, does this fix look right?

Does `--config` still work?

Reply:
> --config is still accepted but is ignored, aside from deprecation notice.

@soltysh that won't work.  Deprecated doesn't mean broken.  The code needs to accept both.

that means it is probably broken elsewhere too.

Reply:
I would actually not be surprised if this is the only place that looks at the flag itself as opposed to the value it sets. And I'd be happy to have it look at the value instead if I could figure out how to retrieve it... digging.